packages/jest-preset-mc-app/transform-babel-jest.js

```diff
@@ -1,6 +1,6 @@
 const getBabePresetConfigForMcApp = require('@commercetools-frontend/babel-preset-mc-app');
 const getJestBabelPreset = require('babel-preset-jest');
-const babelJest = require('babel-jest');
+const { default: babelJest } = require('babel-jest');
```

Comment:
This is likely breaking. If consumers don't update their Jest this won't work. Maybe we have to sniff out the babel-jest version from the `package.json`. Not super sure yet.